packages/apptive_grid_core/lib/src/network/environment.dart

```diff
@@ -1,10 +1,10 @@
 /// Possible Stages/Environment ApptiveGrid can point to
 enum ApptiveGridEnvironment {
   /// Alpha Environment
-  alpha(url: 'https://alpha.apptivegrid.de'),
+  alpha(url: 'http://alpha.apptivegrid.de'),
 
   /// Beta Environment
-  beta(url: 'https://beta.apptivegrid.de'),
+  beta(url: 'http://beta.apptivegrid.de'),
```

Comment:
Did the ssl certificates run out again? Does this break the functionality on android?

Reply:
I just noticed that, because the apptive launcher wasn't parsing the signature type test from uri. I'll ask, if this is permanent or just an oversight

Reply:
reverted